Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -50,7 +50,7 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
         case .loginErrorNotifications:
             return true
         case .paymentsHubMenuSection:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```

Comment:
After the switch, the new feature works as expected 👍 Leaving this PR un-approved for the moment so we can test further.